observe/observe_test.js

```diff
@@ -1384,4 +1384,28 @@ steal('can/util', "can/observe", 'can/map', 'can/list', "can/test", "steal-qunit
 		equal(person._bindings, 0, "After unbinding no bindings");
 	});
 
+	test('compute bound to object property (#1719)', 5, function () {
+		var obj = {};
+		obj.foo = 'bar';
+
+		var value = can.compute(obj, 'foo', 'change');
+		equal(value(), 'bar', 'property retrieved correctly');
+
+		value('baz');
+		equal(obj.foo, 'baz', 'property changed correctly');
+
+		var handler = function(ev, newVal, oldVal) {
+			equal(newVal, 'qux', 'change handler newVal correct');
+			equal(oldVal, 'baz', 'change handler oldVal correct');
+		}
+		value.bind('change', handler);
+		value('qux');
+		value.unbind('change', handler);
+		stop();
+		setTimeout(function() {
+			obj.foo = 'norf';
+			equal(value(), 'norf', 'property changed correctly out of call stack');
+			start();
+		}, 50);
```

Comment:
is this setTimeout necessary?  Also, it 50ms is a long time when there are 1000 tests.